packages/cli/src/commands/aws/vpc.ts

```diff
@@ -0,0 +1,40 @@
+import { ensureFileExists } from "../../util/ensure-file-exists";
+import { replaceHclValue } from "../../util/replace-hcl-value";
+import { apply } from "../terragrunt/apply";
+import { initModules } from "../terragrunt/init-modules";
+import type { BaseContext } from "clipanion";
+
+export interface VpcSetupInput {
+  context: BaseContext;
+  vpcName: string;
+  vpcDescription: string;
+  verbose?: boolean;
+}
+
+export async function setupVpc(input: VpcSetupInput) {
+  await ensureFileExists({
+    context: input.context,
+    downloadUrl: `https://raw.githubusercontent.com/Panfactum/stack/refs/tags/edge.25-03-04/packages/reference/environments/production/us-east-2/aws_vpc/terragrunt.hcl`,
```

Comment:
Created a ticket for this to pull the version from the `environment.yaml` `pf_stack_version` dynamically.